osf_tests/test_node.py

```diff
@@ -3888,6 +3904,36 @@ def test_update_share_correctly_for_projects(self, requests, node, user, request
             graph = kwargs['json']['data']['attributes']['data']['@graph']
             assert graph[1]['is_deleted'] == case['is_deleted']
 
+    @mock.patch('website.project.tasks.settings.SHARE_URL', 'https://share.osf.io')
+    @mock.patch('website.project.tasks.settings.SHARE_API_TOKEN', 'Token')
+    @mock.patch('website.project.tasks.requests')
+    def test_update_collection_elasticsearch_deleted(self, requests, node_in_collection, collection, user, request_context):
+        now = datetime.datetime.now()
+
+        node_in_collection.deleted = now
+
+        on_node_updated(node_in_collection._id, user._id, False, {'deleted'})
+
+        kwargs = requests.post.call_args[1]
+        graph = kwargs['json']['data']['attributes']['data']['@graph']
```

Comment:
Just removing `share` from the test name doesn't mean you're no longer checking the request sent to SHARE. If anything, other than checking ES directly, this should be mocking `website.search.search.update_collected_metadata` and verifying it's called with `op='delete'`.